lib/fog/vsphere/models/compute/rules.rb

```diff
@@ -0,0 +1,28 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Rules < Fog::Collection
+        autoload :Rule, File.expand_path('../rule', __FILE__)
+        
+        model Fog::Compute::Vsphere::Rule
+        attr_accessor :datacenter
+        attr_accessor :cluster
```

Comment:
OK - I've switched the `rules` class to using attributes. You might want to open a task for someone to change all the other collections in fog-vsphere to match.